spec/coverage_helper.rb

```diff
@@ -0,0 +1,14 @@
+if ENV['CI']
+  require 'active_support/core_ext/kernel/reporting'
+
+  # Require all ruby files for accuracte test coverage reports
+  %w(app gems lib spa_ui).each do |path|
```

Comment:
Running on `gems` here is going to ruin the per-job coverage.

Similarly, we should only be doing this for `test:vmdb` (which is nominally full coverage), not the much narrower automation/migrations/replication.